jarbas/dashboard/admin/__init__.py

```diff
@@ -83,6 +85,32 @@ def jarbas(self, obj):
 
     jarbas.short_description = ''
 
+    def social_profile(self, obj):
+        social_media = SocialMedia.objects.filter(congressperson_id=obj.congressperson_id).first()
+        if not social_media:
+            return ''
+
+        tw_link = ''
+        tw_img = '/static/image/twitter-icon.png'
+        if social_media.twitter_profile:
+            tw_link = '<a href="http://twitter.com/{}"><img src="{}" width="16"></a>'.format(
+                social_media.twitter_profile, tw_img
+            )
+        elif social_media.secondary_twitter_profile:
+            tw_link = '<a href="http://twitter.com/{}"><img src="{}" width="16"></a>'.format(
+                social_media.secondary_twitter_profile, tw_img
+            )
```

Comment:
I think a clearer strategy (and DRY) would be something as:

```python
tw_profile = social_media.twitter_profile or social_media.secondary_twitter_profile
if tw_profile:
      tw_link = '<a href="http://twitter.com/{}"><img src="{}" width="16"></a>'.format(
                tw_profile, tw_img
        )
```

Reply:
It's done. Very well observed :wink: